Formula/dnscrypt-proxy.rb

```diff
@@ -144,6 +79,7 @@ def plist; <<~EOS
   end
 
   test do
-    system "#{sbin}/dnscrypt-proxy", "--version"
+    output = shell_output("#{sbin}/dnscrypt-proxy -config #{etc}/dnscrypt-proxy.toml -list 2>&1")
```

Comment:
.................................................................................................

Reply:
You'd like a shorter line?

Reply:
You must have a secret decoder ring.

Reply:
It's my personal "Google Translate" plugin.